Added a Confirmation Message for `Paste Settings` Button

suyalynette opened the pull request:
This is very important to anyone accidentally pressing this button in settings!

arichornlover committed:
Added a Confirmation Message for `Paste Settings` Button

This is very important to anyone accidentally pressing this button in settings!

suyalynette commented:
git checkout -b arichornlover-main main
git pull https://github.com/arichornlover/uYouEnhanced.git main

suyalynette commented:
git checkout main
git merge --no-ff arichornlover-main
git push origin main

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 589.17kb -> 327.64kb (44.39%)

/Bundles/uYouPlus.bundle/AppIcons/Outline.png -- 18.40kb -> 6.92kb (62.37%)
/Localizations/uYouPlus.bundle/AppIcons/Outline.png -- 18.40kb -> 6.92kb (62.37%)
/Bundles/uYouPlus.bundle/youtube_logo_dark.png -- 81.22kb -> 45.20kb (44.35%)
/Localizations/uYouPlus.bundle/youtube_logo_dark.png -- 81.22kb -> 45.20kb (44.35%)
/Localizations/uYouPlus.bundle/youtube_logo.png -- 77.35kb -> 43.44kb (43.84%)
/Bundles/uYouPlus.bundle/youtube_logo.png -- 77.35kb -> 43.44kb (43.84%)
/Localizations/uYouPlus.bundle/AppIcons/White.png -- 5.12kb -> 2.95kb (42.36%)
/Bundles/uYouPlus.bundle/AppIcons/White.png -- 5.12kb -> 2.95kb (42.36%)
/Bundles/uYouPlus.bundle/uYouPlus_logo.png -- 58.31kb -> 33.80kb (42.03%)
/Localizations/uYouPlus.bundle/uYouPlus_logo.png -- 58.31kb -> 33.80kb (42.03%)
/Bundles/uYouPlus.bundle/uYouPlus_logo_dark.png -- 54.19kb -> 31.51kb (41.86%)
/Localizations/uYouPlus.bundle/uYouPlus_logo_dark.png -- 54.19kb -> 31.51kb (41.86%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

Commit:
Merge pull request #1 from SleepingPig35/imgbot

[ImgBot] Optimize images

arichornlover committed:
Merge pull request #641 from SleepingPig35/main

[ImgBot] Optimize images

arichornlover committed:
Added uYou “3.0.1” & “3.0.3” Debs

arichornlover commented:
Just so you know, I was one step ahead 😅
I just now seen this pull request and I was shocked.

the confirmation message is definitely something needed for that button and knowing someone can accidentally press it can be an issue for uYouEnhanced.

gototheskinny committed:
Turkish Loc update for uyouenhanced

TY so much for your work , i appreciate it ❤️

arichornlover committed:
Merge pull request #644 from uyou-icin/main

Turkish Loc update for uyouenhanced.

arichornlover committed:
Added `Copy Settings` Button

arichornlover committed:
Fix `AdBlock Workaround`

This will also disable uYou’s `Remove YouTube Ads` now due to conflicts.

arichornlover committed:
Added New Option `Replace Copy and Paste Buttons`

An Option that alters the behavior of the `Copy Settings` & `Paste Settings` in uYouEnhanced Settings.

arichornlover committed:
Remove duplicates

arichornlover committed:
Add missing line

NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];

arichornlover committed:
💬 New Localization Strings (Template)

arichornlover committed:
Update English Localizable.strings

arichornlover committed:
Update Localizable.strings

arichornlover committed:
Update Localizable.strings
